app/views/course_summaries/_course_summary.html.erb

```diff
@@ -0,0 +1,18 @@
+<% unless @grade_entry_forms.nil? && @assignments.nil? %>
+    <div class='assignment_summary'>
```

Comment:
Always use 2 spaces for indentation (fix the other files too). Also each file should end with exactly one blank line.